storage/api-docs/api-docs.json

```diff
@@ -5944,9 +5952,9 @@
                 "scheme": "https",
                 "flows": {
                     "authorizationCode": {
-                        "authorizationUrl": "http://localhost:8000/oauth/authorize",
-                        "tokenUrl": "http://localhost:8000/oauth/token",
-                        "refreshUrl": "http://localhost:8000/auth/refresh",
+                        "authorizationUrl": "http://localhost:8000/api/v1/oauth/authorize",
+                        "tokenUrl": "http://localhost:8000/api/v1/oauth/token",
+                        "refreshUrl": "http://localhost:8000/api/v1/auth/refresh",
```

Comment:
ah, well. now we have this...

Reply:
@HerrLevin Can you help me here before I break something, you've done?

Reply:
I think that's right?